lib/heartcheck/executors/base.rb

```diff
@@ -0,0 +1,15 @@
+module Heartcheck
+  module Executors
+    class Base
+      def dispatch(checkers)
+        checkers.map do |checker|
+          started = Time.now
+          checker.check.tap do |checked|
+            checked[:time] = (Time.now - started) * 1000.0
+            Logger.info Oj.dump(checked)
```

Comment:
Ops, I can see that you just move the log and not remove like a comment abouve, but with thats way we will log one line for each checker instead of one line for all checkers.

WDYT?